source/javascripts/controllers/_YMLController.js.erb

```diff
@@ -8,6 +8,15 @@ angular.module("BitriseWorkflowEditor").controller("YMLController", function($sc
 	var editor;
 	viewModel.downloadAppConfigYMLPath = requestService.mode == "website" ? requestService.appConfigYMLDownloadPath() : null;
 
+	// discard change detection
```

Comment:
Code should not contain comments unless it is absolutely necessary for some reason. Here it is quite self-explanatory what this watcher does.